app/config.js

```diff
@@ -24,5 +24,9 @@ module.exports = {
 			port: process.env.KAFKA_BROKER_PORT || '9092',
 		},
 	},
-	asyncSchemaUpdates: Boolean(process.env.ASYNC_SCHEMA_UPDATES || 'false'),
```

Comment:
would suggest just `process.env.ASYNC_SCHEMA_UPDATES === 'true'` instead of adding extra function 🤔

Reply:
If you don't like the function, the whole thing could be inlined I suppose. It's nice not to have to worry about casing... How about this instead?

```javascript
asyncSchemaUpdates: (process.env.ASYNC_SCHEMA_UPDATES || 'false').toLowerCase() === 'true',
```

Reply:
@tot-ra Any feedback on the suggestion above?

Reply:
right. thanks, merged & released in 2.2.1, lets see maybe in future PRs can be improved somehow